### DISCUSSION TITLE
Mobile GitHub needs more accessibility.

### DISCUSSION THREAD

**Discussion by xeptao** [community/community#3727] [Mobile] [answered]
I feel like mobile GitHub is just to look at repositories. I can't even edit settings in a repository, the app looks great, but it feels incomplete on UI.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @Xeptao – we definitely aren't fully caught up to support all the functionality of the website, but we're steadily chipping away at features and improvements. We haven't prioritized building the repo settings views on mobile yet, but maybe someday in the future we can get to this. In the meantime, hopefully the website is a good place to manage settings 😊

**Reply by mshll90** [reply]
G

**Comment by xeptao** [comment]
@brianlovin I appreciate your response, respect to the hard-working dev team!

**Comment by Maidaraja** [comment]
Tnx
